uninstall.ps1

```diff
@@ -0,0 +1 @@
+Unregister-ScheduledTask scoop-update-notifier
```

Comment:
In install.ps1, the task name is defined as string. Unify way of definition.

Reply:
Applied in https://github.com/ekishouTV/scoop-update-notifier/pull/1/commits/579c7b852987621a0a92ca3c0c1df25947e3a19d